lib/tests/streamlit/deprecation_util_test.py

```diff
@@ -0,0 +1,121 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
```

Comment:
This is just a renamed version of `beta_util_test.py`. The tests are now parameterized on `PrereleaseAPIType`, so that they test both `beta_` and `experimental_` deprecation warnings.